### DISCUSSION TITLE
Installation for Non Admin Users

### DISCUSSION THREAD

**Discussion by kumkee** [LSPosed/MagiskOnWSALocal#312] [Q&A]
Is it possible to install the compiled app for non-admin users in Windows 11?

I succeeded in installing it as the admin user and it is under the user's AppData folder. Therefore, other users can't use it at all.

How do I share the WSA play store app with other non-admin users?

Or can I install it under their account?

Thanks.

**Comment by cwm9cwm9** [comment]
Same question...?
